app/assets/stylesheets/administrate/components/_field-unit.scss

```diff
@@ -0,0 +1,17 @@
+.field-unit {
```

Comment:
Yeah, it’s tough to find a name for this, but wanted to run with something. I see it as a collective unit of form elements, i.e. a label, input, hint, etc. Could be any number of those as children to this.

Open to suggestions of a better name!